ibm_catalog.json

```diff
@@ -576,6 +757,10 @@
               {
                 "title": "Deploys the IBM Cloud Monitoring agent on an existing cluster on a secure landing zone.",
                 "description": "Deploys the IBM Cloud Monitoring agent on an existing Red Hat OpenShift cluster on a secure landing zone."
+              },
+              {
+                "title": "Deploys the IBM Logs agent on an existing cluster on a secure landing zone.",
```

Comment:
Don't mention secure landing zone - this can deploy on any cluster. I see it in some of the other features, so please remove from those too